docs/source/tutorials/outliers.ipynb

```diff
@@ -516,10 +544,105 @@
+    "\n",
+    "To get `pred_probs` for this example, a Logistic Regression Classifier is fit on the already generated `train_feature_embeddings` from the outlier example above."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "740a8774",
+   "id": "40fed4ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Preprocess data\n",
+    "train_labels = np.array(train_data.dataset.targets)[train_data.indices]\n",
+    "train_labels = np.unique(train_labels, return_inverse=True)[1] # Zero index out train labels\n",
+    "test_labels = np.array(test_data.dataset.targets)[test_data.indices]\n",
+    "\n",
+    "scaler = preprocessing.StandardScaler().fit(train_feature_embeddings)\n",
+    "train_feature_embeddings_scaled = scaler.transform(train_feature_embeddings)\n",
+    "test_feature_embeddings_scaled = scaler.transform(test_feature_embeddings)\n",
+    "\n",
+    "# Create model and fit on scaled features and zero indexed train labels (may take a few minutes)\n",
+    "model = LogisticRegression(penalty='l2', C=1, solver='sag', max_iter=500)\n",
+    "model.fit(train_feature_embeddings_scaled, train_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03e3f7b7",
+   "metadata": {},
+   "source": [
+    "Lets see how well our model fit by checking its accuracy on the train examples. We can also compute out-of-distribution predictions scores for the train dataset using cleanlab's `OutOfDistribution` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e110fc4b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get model predictions and measure accuracy on trainset\n",
+    "train_pred_probs = model.predict_proba(train_feature_embeddings_scaled)\n",
+    "pred_labels = train_pred_probs.argmax(1)\n",
+    "accuracy = np.mean(pred_labels == train_labels)\n",
+    "print(f\"Model fit to train_feature_embeddings accuracy on train_data {accuracy}\")\n",
+    "\n",
+    "# Compute outlier scores for train data\n",
+    "OOD = OutOfDistribution()\n",
+    "train_ood_predictions_scores = OOD.fit(pred_probs=train_pred_probs, labels=train_labels)\n",
+    "\n",
+    "## Visualize top 15 most severe outliers in train data if desired\n",
+    "# top_idxs = (train_ood_predictions_scores).argsort()[:15]\n",
+    "# top_subset = torch.utils.data.Subset(train_data, top_idxs)\n",
+    "# plot_images(top_subset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb631419",
+   "metadata": {},
+   "source": [
+    "Out-of-distribution predictions scores can also be computed on the test data by first calculating `pred_probs` using `test_feature_embeddings`. To do this, an `OutOfDistribution` object fitted on `train_pred_probs` can be used to score `test_pred_probs`.\n",
```

Comment:
The sentence "To do this, ..." is confusing, because it sounds like you are using OutOfDistribution to calculate pred_probs. 

I would edit these sentences to instead read something like this: 
"Out-of-distribution predictions scores can also be computed to estimate how likely additional test data are under the training data distribution. Here we first apply our trained classifier to obtain `pred_probs` for the test data and then pass these `pred_probs` into our already-fitted OutOfDistribution estimator."

Reply:
Side-Note: we really want to minimize mention of the test_feature_embeddings throughout this section, as typically  these would not be relevant for computing pred_probs at all and our application of LogisticRegression to them here is nonstandard and mainly just for convenience.